lib/kennel/models/project.rb

```diff
@@ -11,6 +11,8 @@ class Project < Base
       def self.file_location
         @file_location ||= begin
           method_in_file = instance_methods(false).first
+          return if method_in_file.nil?
```

Comment:
that should never happen now right ?

Reply:
Super edge case, will never happen with normal data. See the test case.